pom.xml

```diff
@@ -2645,7 +2647,18 @@
       <properties>
         <scala.version>2.11.8</scala.version>
         <scala.binary.version>2.11</scala.binary.version>
+        <jline.version>2.12.1</jline.version>
+        <jline.groupid>jline</jline.groupid>
```

Comment:
OK, I think I understand this a little better, but, `repl/pom.xml` no longer needs to manage the jline dependency _only_ for Scala 2.10, right? that can be removed, because it has to be managed (version and group) for both Scala versions.